packages/nodes-base/nodes/HttpRequest/test/utils/utils.test.ts

```diff
@@ -11,15 +11,13 @@ describe('HTTP Node Utils, prepareRequestBody', () => {
 		];
 		const defaultReducer: BodyParametersReducer = jest.fn();
 
-		prepareRequestBody(bodyParameters, 'json', 3, defaultReducer);
+		await prepareRequestBody(bodyParameters, 'json', 3, defaultReducer);
 
 		expect(defaultReducer).toBeCalledTimes(1);
-		expect(defaultReducer).toBeCalledWith({}, { name: 'foo.bar', value: 'baz' }, 0, [
-			{ name: 'foo.bar', value: 'baz' },
-		]);
```

Comment:
It would seem the second pair of args (injected by Jest) are no longer compatible with the async reducer. 

From what I can tell, Jest tracks how many times the function `defaultReducer` has been called, and provides the call index as an argument. And `[{ name: 'foo.bar', value: 'baz' }]` is the list of arguments that `defaultReducer` was called with.

Since these values are already being checked in the first pair of args, to keep this moving I decided to remove the second pair. Let me know if they are needed or you have any insights on why they're incompatible with async.